src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/Regex.Match.Tests.cs

```diff
@@ -316,6 +316,14 @@ private static IEnumerable<(string Pattern, string Input, RegexOptions Options,
 
                 yield return (@".*?", "abc", lineOption, 1, 2, true, "");
                 yield return (@".*?c", "abc", lineOption, 1, 2, true, "bc");
+                yield return (@".*?[^c]", "abc", lineOption, 1, 2, true, "b");
```

Comment:
Probably a dumb question, but why is the expected match for all of these cases `b` instead of the first match `a`?

Reply:
Because of that pesky "1" offset :)

Reply:
oh, gotcha, I thought those were match beginning and end positions, not passed in to Regex.Match